src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss

```diff
@@ -1,116 +0,0 @@
-@import 'vars.scss';
```

Comment:
Why was this deleted?

Reply:
^

Reply:
I had the wrong understanding of how github worked. I thought that this was an extraneous file that made it into my PR by mistake and didn't realize that deleting it from my PR on github would also delete it from the clark-client. I'm fixing it now